### PR TITLE
add local volume 4.11 templates

### PR DESCRIPTION
**Pull request by Bhagyashreek8**

### CODE REVIEW

**Comment by Bhagyashreek8** [comment]
For test results refer this PR - https://github.ibm.com/alchemy-containers/armada-storage-api/pull/4386

File: config-templates/redhat/local-volume-block/4.11/custom-parameters.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "description": "Set to 'true' if you want to automatically discover and use the storage volumes on your worker nodes.",
+    "displayname": "Automatic storage volume discovery",
+    "name": "auto-discover-devices",
+    "default": "false",
```
Comment (mssachan) [inline config-templates/redhat/local-volume-block/4.11/custom-parameters.json:6]: Add regex for this param. Refer https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/redhat/local-volume-block/4.10/custom-parameters.json#L8

File: config-templates/redhat/local-volume-block/4.11/metadata.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "4.11",
+  "revision": "1",
+  "enabled": "true",
+  "status": "supported",
+  "description": "Persistent storage using local volumes",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "redhat"
```
Comment (mssachan) [inline config-templates/redhat/local-volume-block/4.11/metadata.json]: `"redhat"` --> `"Red Hat"`

File: config-templates/redhat/local-volume-file/4.11/metadata.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "4.11",
+  "revision": "1",
+  "enabled": "true",
+  "status": "supported",
+  "description": "Persistent storage using local volumes",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "redhat"
```
Comment (mssachan) [inline config-templates/redhat/local-volume-file/4.11/metadata.json]: `"redhat"` --> `"Red Hat"`

File: config-templates/redhat/local-volume-file/4.11/custom-parameters.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "description": "Set to 'true' if you want to automatically discover and use the storage volumes on your worker nodes.",
+    "displayname": "Automatic storage volume discovery",
+    "name": "auto-discover-devices",
+    "default": "false",
```
Comment (mssachan) [inline config-templates/redhat/local-volume-file/4.11/custom-parameters.json:6]: Add regex for this param. Refer https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/redhat/local-volume-file/4.10/custom-parameters.json#L8

File: config-templates/redhat/local-volume-block/4.11/metadata.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      "value": "For support contact Red Hat"
+    }, {
+      "name": "provider-logo",
+      "value": "logo/local-volume-block.svg"
+    }, {
+      "name": "supported-envs",
+      "value": "AWS,IBM"
+    }, {
+      "name": "supported-k8s-versions",
+      "value": "1.23,1.24,1.25"
```
Comment (mssachan) [inline config-templates/redhat/local-volume-block/4.11/metadata.json]: Lets discuss if we need to have `supported-k8s-versions`

File: config-templates/redhat/local-volume-file/4.11/custom-parameters.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    "place-holder": "/dev/sdc,/dev/sde",
+    "required": "false",
+    "category": "config",
+    "type": "csv"
+  },
+  {
+    "description": "The file system type. Specify 'ext3', 'ext4', or 'xfs'.",
+    "displayname": "File System type",
+    "name": "fstype",
+    "default": "ext4",
```
Comment (mssachan) [inline config-templates/redhat/local-volume-file/4.11/custom-parameters.json:42]: Add regex for this param. Refer
https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/redhat/local-volume-file/4.10/custom-parameters.json#L42

**Review by mssachan — CHANGES_REQUESTED** [review]
PR looks good with minor comments

File: config-templates/redhat/local-volume-file/4.11/custom-parameters.json [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  {
+    "description": "Set to 'true' if you want to automatically discover and use the storage volumes on your worker nodes.",
+    "displayname": "Automatic storage volume discovery",
+    "name": "auto-discover-devices",
+    "default": "false",
+    "type": "boolean",
+    "regex": "^(?:true|false)$",
+    "min-length": "4",
+    "max-length": "5",
+    "required": "false",
```
Comment (mssachan) [inline config-templates/redhat/local-volume-file/4.11/custom-parameters.json:12]: Add `"supported-values": ["true", "false"],` as well

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**